cmd/user_test.go

```diff
@@ -0,0 +1,19 @@
+package cmd
+
+import (
+	"testing"
+	testUtil "github.com/astronomer/astro-cli/pkg/testing"
+	"github.com/astronomer/astro-cli/config"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUserRootCommand(t *testing.T) {
+	fs := afero.NewMemMapFs()
+	configYaml := testUtil.NewTestConfig()
+	err := afero.WriteFile(fs, config.HomeConfigFile, []byte(configYaml), 0777)
+	config.InitConfig(fs)
```

Comment:
These 4 lines appear in all of our tests.

Do you think it's worth collecting these in to a single fn in testUtil, say:

```go
func InitConfigForTests() {
	fs := afero.NewMemMapFs()
	configYaml := NewTestConfig()
	err := afero.WriteFile(fs, config.HomeConfigFile, []byte(configYaml), 0777)
	config.InitConfig(fs)
	if err != nil {
		panic(err); // Shouldn't happen
	}
}
```